lib/daemon.js

```diff
@@ -28,7 +28,9 @@ export default function daemon() {
       process.stdout.write(`${JSON.stringify(result)}\n`);
     }).catch((error) => {
       // TODO: print entire stack trace here
-      process.stderr.write(error.toString());
+      process.stdout.write(JSON.stringify({
```

Comment:
If I remember correctly, I needed this to make communicating with the daemon in vim simpler. It also makes communication in the sublime plugin simpler, because I can wait for a line to be written to stdout.